packages/plugins/rubygem-hammer_cli_foreman_openscap/rubygem-hammer_cli_foreman_openscap.spec

```diff
@@ -1,70 +1,74 @@
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-
+# template: hammer_plugin
 %global gem_name hammer_cli_foreman_openscap
-%global confdir hammer
+%global plugin_name foreman_openscap
 
-Summary: Foreman OpenSCAP commands for Hammer CLI
-Name: %{?scl_prefix}rubygem-%{gem_name}
+%global hammer_confdir %{_sysconfdir}/hammer
+
+Name: rubygem-%{gem_name}
 Version: 0.1.13
-Release: 1%{?foremandist}%{?dist}
-Group: Applications/System
-License: GPLv3
-URL: https://github.com/theforeman/hammer_cli_foreman_openscap
+Release: 2%{?foremandist}%{?dist}
+Summary: Foreman OpenSCAP commands for Hammer
+License: GPL-3
```

Comment:
`GPL-3` doesn't look correct?

Reply:
Are you saying it should be `GPL-3.0` upstream to be valid SPDX? Then our [license.json](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/licenses.json) will also understand it.

Reply:
That's what I said and committed to :)

Reply:
Well, GH didn't show your message until I submitted. I blame that.